build/macos/main.applescript

```diff
@@ -0,0 +1,81 @@
+on ensureLineInFileIfExists(filePath, lineToAdd)
+   set ok to false
+   try
+      set fileAlias to POSIX file filePath as alias
+      local fileDescriptor
+      set fileDescriptor to open for access fileAlias with write permission
+      try
+         set lns to paragraphs of (read fileDescriptor)
+         repeat with ln in lns
+            if ln is lineToAdd then
+               set ok to true
+               exit repeat
+            end if
+         end repeat
+         if ok is false then
+            write return & lineToAdd & return to fileDescriptor starting at eof
+            set ok to true
+         end if
+      end try
+      close access fileDescriptor
+   end try
+   return ok
+end findLineInFile
+
+on setupEnvironment(binFolder, binPath, launchAgentName)
+   set homeFolder to POSIX path of (path to home folder)
+
+   set bashProfilePath to homeFolder & ".bash_profile"
+   set zshrcPath to homeFolder & ".zshrc"
+   set exportString to "export PATH=" & quote & binFolder & ":$PATH" & quote & " # Added by Spicetify"
+
+   ensureLineInFileIfExists(bashProfilePath, exportString)
+   ensureLineInFileIfExists(zshrcPath, exportString)
+
+   set plistPathQ to quoted form of (homeFolder & "/Library/LaunchAgents/" & launchAgentName & ".plist")
```

Comment:
per DMs, this doesn't work
![image](https://github.com/user-attachments/assets/fecf4768-f05a-4fab-b2b8-8f503d279bb6)

Reply:
This additional `/` doesn't resolve this issue btw

Reply:
already fixed
![image](https://github.com/user-attachments/assets/d62f527d-5688-4686-b075-cc98a6bb73fc)